native/ios/Podfile.lock

```diff
@@ -286,8 +286,6 @@ PODS:
   - react-native-ffmpeg/min-lts (0.4.4):
     - mobile-ffmpeg-min (= 4.3.1.LTS)
     - React
-  - react-native-in-app-message (1.0.2):
```

Comment:
This is strange - why is `react-native-in-app-message` removed here?

Reply:
I was also confused, but it happened after I did `pushd native && yarn remove react-native-hyperlink && popd && yarn cleaninstall`

Reply:
Some weird bug I guess, I had to run `yarn clean install` couple of times and it's back now